v3/src/components/graph/v2-graph-exporter.test.ts

```diff
@@ -157,4 +157,22 @@ describe("V2GraphImporter", () => {
       expect(v2GraphTileOutStorage).toEqual(v2GraphTileStorage)
     })
   })
+
+  it("exports graph components with LSLR adornments", () => {
+    const { v2Document } = loadCodapDocument("mammals-lsrl-adornment.codap")
+    const v2GraphTiles = v2Document.components.filter(c => c.type === "DG.GraphView")
+    v2GraphTiles.forEach(v2GraphTile => {
+      // console.log("v2GraphTile", v2GraphTile.componentStorage.title || v2GraphTile.componentStorage.name)
```

Comment:
remove console.log

Reply:
I'm intentionally leaving those in (but commented out) because they're useful for determining which graph is failing when the test fails, so during development uncommenting those console.logs is often the first step.